build-info-extractor-gradle/src/main/groovy/org/jfrog/gradle/plugin/artifactory/extractor/listener/ProjectsEvaluatedBuildListener.groovy

```diff
@@ -108,13 +109,27 @@ public class ProjectsEvaluatedBuildListener extends BuildAdapter implements Proj
             if (resolver != null) {
                 defineResolvers(artifactoryTask.project, resolver)
             }
-            if (artifactoryTask.isCiServerBuild()) {
+            if (artifactoryTask.isCiServerBuild() && isJava(artifactoryTask)) {
                 addDefaultPublicationsOrConfigurations(artifactoryTask);
             }
             artifactoryTask.projectEvaluated()
         }
     }
 
+    /**
+     * Return true if the 'java' or 'java-library' plugins are applied.
+     * @param artifactoryTask - The artifactory task
+     * @return true if the 'java' or 'java-library' plugins are applied.
+     */
+    private static boolean isJava(ArtifactoryTask artifactoryTask) {
```

Comment:
I think it makes more sense of this method ro receive a *Project* as an argument, rather than a task.